gromacs/config.py

```diff
@@ -266,50 +228,36 @@
 """
 from __future__ import absolute_import, with_statement
 
-import os, errno, subprocess
-from ConfigParser import SafeConfigParser
+import os
+import logging
+import subprocess
 
+from ConfigParser import SafeConfigParser
 from pkg_resources import resource_filename, resource_listdir
 
 from . import utilities
 
-# Defaults
-# --------
-# hard-coded package defaults
 
-#: name of the global configuration file.
+# Default name of the global configuration file.
 CONFIGNAME = os.path.expanduser(os.path.join("~",".gromacswrapper.cfg"))
 
-#: Holds the default values for important file and directory locations.
-#:
-#: :data:`configdir`
-#:    Directory to store user templates and configurations.
-#:    The default value is ``~/.gromacswrapper``.
-#: :data:`qscriptdir`
-#:    Directory to store user supplied queuing system scripts as
-#:    used by :mod:`gromacs.qsub`.
-#:    The default value is ``~/.gromacswrapper/qscripts``.
-#: :data:`templatesdir`
-#:    Directory to store user supplied template files such as mdp files.
-#:    The default value is ``~/.gromacswrapper/templates``.
-#: :data:`managerdir`
-#:    Directory to store configuration files for different queuing system
-#:    managers as used in :mod:`gromacs.manager`.
-#:    The default value is ``~/.gromacswrapper/managers``.
+# More default values
```

Comment:
Why did you remove the docs? They were in the generated docs and the information is not available elsewhere, I think.